tools/autograd/gen_variable_type.py

```diff
@@ -348,7 +348,7 @@
 GRADIENT_IMPLEMENTED_FOR_COMPLEX.update(GRADIENT_IMPLEMENTED_FOR_SPARSE_COMPLEX)
 
 # Some operators invalidate the grad_accumulator. Let's reset it.
-RESET_GRAD_ACCUMULATOR = {"set", "resize"}
+RESET_GRAD_ACCUMULATOR = {"set_", "resize_"}
```

Comment:
Needed to make these checks more specific now that there's a `set.functional` and `resize.functional` NativeFunction.